c++/demos/README.md

```diff
@@ -1,125 +1,89 @@
-# Qeep-SDK-Demo-C-App
+# Qeep SDK Demo for C++ Application
 
-A functionality demo based in C++ to demonstrate the usage of the QEEP, SEQUR, PQRND and MASQ APIs as well as QiSpace RESTful API. For the best running experience, run the demo on an Ubuntu computer.
+A functionality demo based in C++ to demonstrate the usage of the MASQ APIs. For the best running experience, run the demo on Ubuntu platform.
 
-## **Overview:**
-- **SEQUR**: Demonstrates the use of **SEQUR API to decode arbitrary QiSpace Enterprise payloads** using either a pre-loaded or QiSpace-REST-API-retrieved subscriber key (subkey).
-- **QEEP**: Demonstrates use of **QEEP API for encryption/decryption** and, if desired, the process of retrieving a Quantum Key (QK) from QiSpace Enterprise. 
-- **PQRND**: Demonstrates use of **PQRND API for generating Pseudo Quantum Random Data** with either pre-loaded Quantum Entropy (QE) or QiSpace-REST-API-retrieved QE.
-- **MASQ**: Provides **Asymmetric Encryption** functionality. The two main functions of MASQ are **Key Exchange and Digital Signature** using a pair of public and private keys.
+## **Overview**
+- **MASQ** provides **Asymmetric Encryption** functionality. The two main functions of MASQ are **Key Exchange** and **Digital Signature** using a pair of public and private keys.
 
-## **Installation and Preparation:**
-1. Download or checkout this git repository.
-2. Place the `libmasq`, `libqeep` folders containing MASQ and QEEP libraries for your OS under this repository's `(/c/demos)` folder.
-    - `libmasq`: should contain all the contents from `ds` and `kem` from QiSpace MASQ SDK .`ds` and `kem` contain the needed Big Number libraries.
-    - `libqeep`: should contain contents from `lib` folder under QiSpace QEEP SDK.
-
-    The `libmasq` folder will look like below:
-  
-    ![libmasq, the libmasq folder](libmasq.png)
-
-    The `libqeep` folder will look like below:
+## **Installation and Preparation**
+1. Download or checkout `qispace-sdk-code-sample` git repository to the local machine.
+2. Contact Quantropi to get the latest QiSpace MASQ SDK libraries.
+3. Create the `libmasq` folder containing MASQ libraries depending on the platform at the top of this repository. The `libmasq` should contain all the contents from `ds` and `kem` from QiSpace MASQ SDK. The `ds` and `kem` contain the needed Big Number libraries.
```

Comment:
We can break down this:
3. On the same folder where the downloaded `git repo demo` was saved, create `libmasq` folder.
5. On the created `libmasq` folder, copy all the contents of `ds` and `kem` folder from QiSpace MASQ SDK librarries according to the platform where the demo will be run. Note that the QiSpace MASQ SDK libraries includes the Big Number libraries as well.